WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostListViewModel.kt

```diff
@@ -88,6 +97,9 @@ class ReaderPostListViewModel @Inject constructor(
     private val _updateTagsAndSites = MutableLiveData<Event<EnumSet<UpdateTask>>>()
     val updateTagsAndSites: LiveData<Event<EnumSet<UpdateTask>>> = _updateTagsAndSites
 
+    private val _reblogAction = MutableLiveData<Event<ReblogState>>()
+    val reblogAction: LiveData<Event<ReblogState>> = _reblogAction
```

Comment:
Since it is a state, maybe we could name it `_reblogState`? wdyt?

Reply:
I was in a dilemma on this. It's a state but also an event/action.
I will agree with you and change the name.
Fixed 7743337c0c59a0e22c21c982a28fa87176f5d935